stable/minio/values.yaml

```diff
@@ -3,7 +3,7 @@
 ## Distributed Minio ref: https://docs.minio.io/docs/distributed-minio-quickstart-guide
 ##
 image: "minio/minio"
-imageTag: "RELEASE.2018-01-18T20-33-21Z"
+imageTag: "RELEASE.2018-02-09T22-40-05Z"
 imagePullPolicy: "Always"
```

Comment:
Could you change `imagePullPolicy` to `IfNotPresent`?

Reply:
good point @unguiculus but I wanted to keep this as `Always` just to make sure all the deployments run the same and latest Minio release. 

Given the Minio image is just 11MB, IMHO this is not a very large load on users

Reply:
Always only makes sense for `latest` unless you update an existing tag.

Reply:
> Always only makes sense for latest 

I did not get @unguiculus could you please elaborate..

Reply:
Always doesn't make sense for stable image that don't change. There's no point in always trying to pull the image again. Please change it to `IfNotPresent`.